### PR TITLE
[DOC][New files] setup.md, prerequisites.md, contributing.md

### PR DESCRIPTION
**Pull request by fartzy**
Summary - Adding these three files (Setup.md, export.md, import.md) in the `docs/` folder which will be the main doc files.  The root README.md will still be the first doc file that a user will see. 

Still need to have root README.md link to these three files.  Also README.md has some empty sections in the bottom of it currently that need to be incorporated into these three files.

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+# Databricks-Sync
+
+Databricks-Sync is for workspace disaster recovery. You can use it to import or export some, or all, objects within a workspace. Databricks-Sync can also be used as a migration utility.
+
+## Dependencies
+
+This project requires the following environment dependencies:
+
+* [golang 1.13.x](https://golang.org/dl/)
```
Comment (itaiw) [inline docs/setup.md:17]: @fartzy we should remove Go from here and in step #2 
We do not need it anymore

File: docs/setup.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* [terraform 0.12.x](https://www.terraform.io/downloads.html)
+* [terraform-provider-databricks 0.2.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
+
+## Installation
+
+0. Download: `$ git clone https://github.com/databrickslabs/terraform-provider-databricks.git`
+1. Install Databricks Terraform provider: `$ curl https://raw.githubusercontent.com/databrickslabs/databricks-terraform/master/godownloader-databricks-provider.sh | bash -s -- -b $HOME/.terraform.d/plugins`
```
Comment (itaiw) [inline docs/setup.md:24]: We should move it to another section, this is needed only if they are on terraform 0.12, or do not have internet access

File: docs/setup.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    1. Verify `terraform` version 0.12.x: `$ terraform version`
+    2. Verify `golang` version 1.13.x: `$ go version`
+    3. Verify `make` version 3.81: `$ make --version`
+3. Install other dependencies: `$ cd terraform-provider-databricks && pip install -r requirements.txt`
+4. Install this project: `$ make shared install`
+5. Test that the installation succeeded: `databricks-terraformer --version`
+
+## Authentication
+
+### [Databricks-CLI](https://docs.databricks.com/dev-tools/cli/index.html)
```
Comment (itaiw) [inline docs/setup.md:35]: Can we move the CLI section to a Pre-requisite?
I think it'll make more sense and some can skip it if it's already setup for them

File: docs/setup.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  * `--git-ssh-url` or `-g` - The URL of the git repo should look like `git@github.com:USERNAME/REPOSITORY.git`
+  * `--local-git-path` or `-l` - The path of a local git repo `/path/to/local/git/repo`
+* `--branch` (Optional) This is the git repo branch of the repo designated by `--git-ssh-url flag | --local-git-path`. If not given, the default branch is `master`.
+* `--ssh-key-path` or `-k` - (Optional) CLI connection profile to use. The default value is "~/.ssh/id_rsa". This is equivalent to the `-i` switch when using `ssh`.
+* `--verbosity` or `-v` - (Optional) For logging, takes a value of `DEBUG`, `INFO`, `WARNING`, `ERROR`, or `CRITICAL`.
+* `--version` - (Optional) A version can be attached.
+
+### Export - arguments for export
+
+* `--config-path` or `-c` - This is the relative path (to the root directory of this repo) or the full path of the yaml file which is used to drive which bjects are imported/exported.
```
Comment (itaiw) [inline docs/setup.md:66]: need to add --branch in here to support github new "main"

File: docs/setup.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+| --: | --- |
+| `--profile` | `AZURE_SOURCE_WORKSPACE` or `AZURE_TARGET_WORKSPACE` |
+| `--git-ssh-url` | `GIT_REPO` |
+| `--branch` | `MASTER_REVISION` |
+| `--revision` | `MASTER_REVISION` |
+| `--destroy` | `DESTROY` |
+| `--artifact-dir` | `ARTIFACT_DIR` |
+| `--backup` | `BACKUP_FILE` |
+
+### Using .env file
```
Comment (itaiw) [inline docs/setup.md:97]: This should probably move tot he Contributing.md file

**Review by itaiw — CHANGES_REQUESTED** [review]
Great start. Please modify the file per inline comments

File: docs/setup.md (hunk 1 above)
Comment (fartzy) [reply docs/setup.md:17]: Ok I took golang out

File: docs/setup.md (hunk 2 above)
Comment (fartzy) [reply docs/setup.md:24]: I created prerequisites.md in same docs directory

File: docs/setup.md (hunk 3 above)
Comment (fartzy) [reply docs/setup.md:35]: I moved this also toe prerequisites.md

File: docs/setup.md (hunk 5 above)
Comment (fartzy) [reply docs/setup.md:97]: I created contributing.md and added this to it.

File: docs/setup.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Prerequisites
+
+Visit the [prerequisites page](prerequisites.md) to make sure you have all the prerequisites before beginning installation.
+
+## Dependencies
+
+This project requires the following environment dependencies:
+
+* [terraform 0.12.x](https://www.terraform.io/downloads.html)
```
Comment (itaiw) [inline docs/setup.md:21]: fix version to 0.13

**Review by itaiw — APPROVED** [review]

File: docs/setup.md (hunk 6 above)
Comment (fartzy) [reply docs/setup.md]: fixed

File: docs/setup.md (hunk 5 above)
Comment (fartzy) [reply docs/setup.md]: fixed

File: docs/setup.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+* [terraform-provider-databricks 0.2.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
+
+## Installation
+
+1. Ensure Terraform and Golang dependencies (install via the package manager of your choice):
```
[the remark this answers is not in the document]
Comment (fartzy) [reply docs/setup.md]: removed

**Pull request merged**